packages/test-runner/src/dispatcher.ts

```diff
@@ -371,8 +392,20 @@ class InProcessWorker extends Worker {
   }
 }
 
-function chunkFromParams(params: { testId: string, buffer?: string, text?: string }): string | Buffer {
+function chunkFromParams(params: TestOutputPayload): string | Buffer {
   if (typeof params.text === 'string')
     return params.text;
   return Buffer.from(params.buffer, 'base64');
 }
+
+function createTestResult(retryNumber: number): TestResult {
+  return {
+    retryNumber: retryNumber,
+    workerIndex: 0,
+    duration: 0,
+    status: 'passed',
```

Comment:
Needs to be either `'none'` or `undefined` for shards logic - some tests don't run at all.